.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: cpp
+env:
+  global:
+  - FORCE_INSTALL=1
```

Comment:
You do not need `env` part at all, it is for `macbuild tool`. Are there any issues? If not, I can merge.